apps/prairielearn/src/lib/assets.js

```diff
@@ -20,11 +20,11 @@ async function hashDirectory(dir) {
 }
 
 async function computeAssetsHash() {
-  assetsHash = await hashDirectory(path.join(__dirname, '..', 'public'));
+  assetsHash = await hashDirectory(path.join(__dirname, '..', '..', 'public'));
```

Comment:
@mwest1066 I'm not planning on ever moving this code again, but I do wonder if it'd be nice to encapsulate this with something like `PACKAGE_ROOT` and `REPOSITORY_ROOT` constants that we can define in a single place? That way we don't have to carefully count the correct number of `..` every time we want to access something at a given path?

Reply:
That feels helpful even just from a code readability point of view.

Reply:
This was done in #7665.